eclass/check-reqs.eclass

```diff
@@ -192,15 +202,16 @@ check-reqs_get_number() {
 	# Check for unset units and warn about them.
 	# Backcompat.
 	if [[ ${size} == ${1} ]]; then
-		ewarn "QA: Package does not specify unit for the size check"
-		ewarn "QA: Assuming mebibytes."
-		ewarn "QA: File bug against the package. It should specify the unit."
+		eqawarn "Package does not specify unit for the size check"
+		eqawarn "Assuming mebibytes."
+		eqawarn "File bug against the package. It should specify the unit."
 	fi
```

Comment:
In fact, nonfatal does not make any sense here. This is only called in combination with _get_unit, which is fatal in this circumstance.